.prettierignore

```diff
@@ -6,3 +6,4 @@ lcc-react-js-bike-model/
 **/staticresources/**
 .localdevserver
 .sfdx
+.vscode
```

Comment:
Good addition. Please add that to the other sample apps or create issues on them that we add this consistently.

Reply:
Yes, I opened issues on all repos.